pubspec.yaml

```diff
@@ -3,64 +3,89 @@ description: Port of https://github.com/PolymerLabs/polymer-snippets
-    - web/forms/binding_to_a_textarea/index.html
-    - web/forms/implementing_simple_autocompletion/index.html
-    - web/forms/implementing_simple_validation/index.html
-    - web/forms/selecting_one_item_using_radio_buttons/index.html
-    - web/forms/selecting_many_items_using_checkboxes/index.html
+   # - web/forms/binding_a_boolean_field_to_a_checkbox/index.html
+   # - web/forms/binding_to_a_text_input/index.html
+   # - web/forms/binding_to_a_textarea/index.html
+   # - web/forms/implementing_simple_autocompletion/index.html
+   # - web/forms/implementing_simple_validation/index.html
+   # - web/forms/selecting_one_item_using_radio_buttons/index.html
+   # - web/forms/selecting_many_items_using_checkboxes/index.html
 
-    - web/events/using_custom_events/index.html
+   # - web/events/using_custom_events/index.html
 
-    - web/styling_elements/applying_styles_by_piercing_shadow_dom_boundaries/index.html
-    - web/styling_elements/defining_styles_inside_a_polymer_element/index.html
-    - web/styling_elements/programmatically_changing_the_styles_of_an_element/index.html
-    - web/styling_elements/styling_distributed_nodes/index.html
-    - web/styling_elements/using_host_context_to_theme_an_element/index.html
-    - web/styling_elements/using_host_with_a_css_selector/index.html
-    - web/styling_elements/using_host_with_pseudo_classes/index.html
+   # - web/styling_elements/applying_styles_by_piercing_shadow_dom_boundaries/index.html
+   # - web/styling_elements/defining_styles_inside_a_polymer_element/index.html
+   # - web/styling_elements/programmatically_changing_the_styles_of_an_element/index.html
+   # - web/styling_elements/styling_distributed_nodes/index.html
+   # - web/styling_elements/using_host_context_to_theme_an_element/index.html
+   # - web/styling_elements/using_host_with_a_css_selector/index.html
+   # - web/styling_elements/using_host_with_pseudo_classes/index.html
 
     - web/layout_attributes/nesting_flex_layouts/index.html
     - web/layout_attributes/using_flex/index.html
     - web/layout_attributes/using_layout_attributes/index.html
 
-    - web/core_elements/core_toolbar/adding_a_menu_button/index.html
-    - web/core_elements/core_toolbar/adding_button_rows/index.html
-    - web/core_elements/core_toolbar/changing_the_toolbar_size/index.html
-    - web/core_elements/core_toolbar/using_core_toolbar/index.html
+    - web/polymer_elements/paper_toolbar/adding_a_menu_button/index.html
+    - web/polymer_elements/paper_toolbar/adding_button_rows/index.html
+    - web/polymer_elements/paper_toolbar/changing_the_toolbar_size/index.html
+    - web/polymer_elements/paper_toolbar/using_paper_toolbar/index.html
+
+- reflectable:
+    entry_points:
+    - web/layout_attributes/nesting_flex_layouts/index.dart
+    - web/layout_attributes/using_flex/index.dart
+    - web/layout_attributes/using_layout_attributes/index.dart
+
+    - web/paper_elements/paper_toolbar/adding_a_menu_button/index.html
```

Comment:
If an entry point has zero Polymer Elements or JsProxy classes then  you actually don't need the reflectable transformer at all.